tools/ratbagctl.in

```diff
@@ -1468,7 +1468,7 @@ def main(argv):
         parser.print_help()
         return
 
-    r = open_ratbagd()
+    r = open_ratbagd(cmd.verbose)
```

Comment:
the verbose commits seem like detritus

Reply:
Actually no (for ratabgd.py, it's unused, I concede). It is used in the ratbagc python wrapper to set the verbose mode too here.

I guess this should eventually be a separate commit.